### ISSUE TITLE
Project naming

### ISSUE THREAD

**Issue by g-bon** [b3nab/instapy-cli#1]
Hi, thanks for setting up this new fork. This is to be intended as a suggestion and not an issue.

Reflecting on the name, I can see there is an existing instapy project with quite a lot of stars. I think this being a fork from pynstagran and being named so close to instapy can create some confusion.

My suggestion is to find a name that is closer to the project is forked from, or a completely new one.

Looking forward to contribute to this.

**Comment by b3nab** [comment]
Yes, definitely. 
Now I'm focusing on a possible way to improve and make this cli more configurable and extendable.

However some alternative name could be:
- instapyload
- instaloader
-instaload

My suggestion is to change the name only one time. 
So I keep this issue open to suggestions.

**Comment by b3nab** [comment]
Other suggestion (playing with assonance of the word):

- instaup
- instup
- instauppy
- instuppy
- instupy

Just to name it correctly with the purpouse of the package. 
"insta" is for Instagram (of course..), "up" is for upload and "py" is for python, the programming language used for core of this package. But I think it could be omitted.

The bests match for this could be instaup or instup.

**Comment by g-bon** [comment]
I agree, my favorite is instaup. And I'm super happy to see a python package that doesn't contain "py".

**Comment by b3nab** [comment]
And for example, if we need to change the name removing the prefix "insta-" do you suggest something?

**Comment by DyingEcho** [comment]
`pynsta`?

**Comment by b3nab** [comment]
Sounds good, and there isn't a package on pypi named this way.
But, maybe is it a little bit generic? 
For now this client is made only to upload a photo to instagram.

**Comment by vitorgalvao** [comment]
> I search for a good, stable and maintained library for unofficial private instagram api

have you looked at [Instagram-API-python](https://github.com/LevPasha/Instagram-API-python)?

**Comment by b3nab** [comment]
@vitorgalvao  maybe that could be a valid alternative to "reinvent-the-wheel".
I'll check it out and write here when I have some news.

**Comment by b3nab** [comment]
Maybe now, from v0.0.7 have some meaning to call this library pynsta, due to moving the Instagram session management from internal (on api/session.py file) to external library (via [instagram-private-api](https://github.com/ping/instagram_private_api)).

This will let us to do some great stuffs from terminal, and there is a need to design a "standard" commands and options to use with instapy-cli.

Other news will come.. 🥃

**Comment by b3nab** [comment]
## We are moving!
**instapy-cli** is making its way in the instagram-related tools.
Some days ago I talked with @ohld and we decide to join **instapy-cli** under the @instagrambot/instagram-cli team. 

In the future we will make some cool project, based on **instapy-cli** and on the other libraries under the instagrambot Organization. 

Stay tuned! 🎉
